server/app/views/applicant/EditTiClientView.java

```diff
@@ -95,20 +100,43 @@ private DivTag renderEditClientForm(
       Http.Request request,
       Optional<Form<EditTiClientInfoForm>> form,
       Messages messages) {
-    ApplicantData applicantData = account.newestApplicant().get().getApplicantData();
+    ApplicantModel applicant = account.newestApplicant().get();
+    ApplicantData applicantData = applicant.getApplicantData();
     FormTag formTag =
         form()
             .withId("edit-ti")
             .withMethod("POST")
             .withAction(routes.TrustedIntermediaryController.updateClientInfo(account.id).url());
+    // TODO (#5503): Remove using WKP functions when removing feature flag
+    Optional<String> firstName =
+        settingsManifest.getPrimaryApplicantInfoQuestionsEnabled(request)
+            ? applicant.getFirstName()
```

Comment:
question (maybe blocking): do we need to check that this is present and use `.get` before setting it as a value below?